lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -40,6 +40,16 @@ static LogicalResult verifyProgramOp(ProgramOp program) {
 // ComponentOp
 //===----------------------------------------------------------------------===//
 
+/// Gets the WiresOp of a component.
+static WiresOp getWiresOp(ComponentOp componentOp) {
+  WiresOp wiresOp;
+  auto body = componentOp.getBody();
+  auto opIt = llvm::find_if(*body, [](auto &&op) { return isa<WiresOp>(op); });
+
+  assert(opIt != body->end() && "A component should have a WiresOp.");
+  return dyn_cast<WiresOp>(*opIt);
```

Comment:
This can be cast instead of dyn_cast since it will always succeed.

Reply:
Thanks!